Python/Setup/PythonTools/PythonToolsFiles.proj

```diff
@@ -135,7 +136,11 @@
     <File Include="$(BuildOutputRoot)Binaries\Analyzer\CompletionDB\*.idb">
       <InstallDirectory>CompletionDB</InstallDirectory>
     </File>
-  
+
+    <File Include="*.idb">
+      <InstallDirectory>CompletionDB</InstallDirectory>
+    </File>
+    

```

Comment:
Yes, these are analyzed offline since it has to be done inside of an app to get these libraries updated, but I should probably remove this since it was a stop-gap to have intellisense for these libraries.  They should/would be included with whatever package pulled them in.